Rubeus/Commands/ASREP2Kirbi.cs

```diff
@@ -86,7 +86,16 @@ public void Execute(Dictionary<string, string> arguments)
             }
             else if (arguments.ContainsKey("/keyhex"))
             {
-                key = Helpers.StringToByteArray(arguments["/keyhex"]);
+                try
+                {
+                    key = Helpers.StringToByteArray(arguments["/keyhex"]);
```

Comment:
Instead of putting the try/catch around the single function call, would it not be cleaner to have a higher level try block in Main for ArgumentException?